lib/rester/client/adapters/stub_adapter.rb

```diff
@@ -31,6 +31,8 @@ def connected?
       end
 
       def request!(verb, path, encoded_data)
+        return [200, {}, []] if verb == :get && path == '/ping'
+
```

Comment:
Ooo, maybe this is what we can use the `producer` key for that's in the stub file, but haven't actually used yet.  That could be used to set the producer header :)